src/rwa/components/table/expandable-row.tsx

```diff
@@ -0,0 +1,34 @@
+import { mergeClassNameProps } from '@/powerhouse';
+import { ComponentPropsWithRef } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+export interface RWATableRowProps extends ComponentPropsWithRef<'tr'> {
```

Comment:
I think you meant to use `ComponentPropsWithoutRef` for this and the `td`props because these compoments don't forward refs